### ISSUE TITLE
Not in scope: type constructor or class ‘Foldable’

### ISSUE THREAD

**Issue by pmlodawski** [atzeus/HMap#2]
Hi! It looks like master version have some compile issues

```
Resolving dependencies...
Notice: installing into a sandbox located at
/home/pmlodawski/dev/projects/HMap/.cabal-sandbox
Configuring text-1.2.1.1...
Configuring transformers-0.4.3.0...
Building text-1.2.1.1...
Building transformers-0.4.3.0...
Installed transformers-0.4.3.0
Configuring mtl-2.2.1...
Building mtl-2.2.1...
Installed mtl-2.2.1
Installed text-1.2.1.1
Configuring hashable-1.2.3.2...
Building hashable-1.2.3.2...
Installed hashable-1.2.3.2
Configuring unordered-containers-0.2.5.1...
Building unordered-containers-0.2.5.1...
Installed unordered-containers-0.2.5.1
Configuring HMap-1.2.3...
Building HMap-1.2.3...
Failed to install HMap-1.2.3
Build log ( /home/pmlodawski/dev/projects/HMap/.cabal-sandbox/logs/HMap-1.2.3.log ):
Configuring HMap-1.2.3...
Building HMap-1.2.3...
Preprocessing library HMap-1.2.3...
[1 of 6] Compiling Data.HideType    ( Data/HideType.hs, dist/dist-sandbox-47edc933/build/Data/HideType.o )
[2 of 6] Compiling Data.HKeyPrivate ( Data/HKeyPrivate.hs, dist/dist-sandbox-47edc933/build/Data/HKeyPrivate.o )
[3 of 6] Compiling Data.Untypeable  ( Data/Untypeable.hs, dist/dist-sandbox-47edc933/build/Data/Untypeable.o )
[4 of 6] Compiling Data.HKey        ( Data/HKey.hs, dist/dist-sandbox-47edc933/build/Data/HKey.o )
[5 of 6] Compiling Data.HMap        ( Data/HMap.hs, dist/dist-sandbox-47edc933/build/Data/HMap.o )
[6 of 6] Compiling Data.HKeySet     ( Data/HKeySet.hs, dist/dist-sandbox-47edc933/build/Data/HKeySet.o )

Data/HKeySet.hs:75:11:
    Not in scope: type constructor or class ‘Foldable’
cabal: Error: some packages failed to install:
HMap-1.2.3 failed during the building phase. The exception was:
ExitFailure 1
```

**Comment by atzeus** [comment]
This is due to you having an older version of GHC I think, but I've fixed it :) See HMap-1.25

**Comment by pmlodawski** [comment]
I cannot see it on this repo... I have also created a pull request fixing this issue: https://github.com/atzeus/HMap/pull/6
